src/pages/home/_Hero.tsx

```diff
@@ -1,14 +1,10 @@
 import { FunctionComponent, useEffect, useState } from 'react'
 
-import classNames from 'classnames'
 import Link from 'next/link'
 
-import { ContentSection, CustomerLogos } from '@components'
+import { ContentSection, CustomerLogos, Video } from '@components'
 import { buttonStyle, buttonLocation } from '@data'
 
-import meshLeft from './assets/hero/mesh-left.png'
-import meshRight from './assets/hero/mesh-right.png'
```

Comment:
Tracey mentioned these were staying for brand recognition, so we may want to double check with her. Also, I was chatting with her the other day about this and I think we should use SVGs instead of PNGs for these.

Reply:
Ok cool cool! Thanks for the update! Excited to see what's coming! ✨

Reply:
They looked massive so I adjusted the size a bit and removed the bootstrap classes